mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +31,41 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Image respond to clicks
+   * * @default false
+   */
+  isZoomable?: boolean;
 }
 
 /**
- * Сomponent for displaying images
+ * Component for displaying images
  */
 export const Image = (props: ImageProps) => {
-  const className = clsx(styles.image, props.className);
+  const imageClass = clsx(styles.image, props.className);
 
-  return (
+  const imageElement = (
     <img
       src={props.src}
       alt={props.alt}
-      className={className}
+      className={imageClass}
       data-cy={props.dataCy}
     />
   );
-};
 
+  if (props.isZoomable) {
+    return (
+      <Modal
+        cy={props.cy}
+        trigger={imageElement}
+        content={imageElement}
+        className={imageClass}
+        data-cy={props.cy}
```

Comment:
What the difference between cy and data-cy props? Delete data-cy attribute

Reply:
in here i removed data-cy because it repeats with cy